environments/base/default-config.libsonnet

```diff
@@ -38,6 +38,7 @@
     retentionResolutionRaw: '14d',
     retentionResolution5m: '1s',
     retentionResolution1h: '1s',
+    replicas: 1,
```

Comment:
It will be used by operator. That is one question: Do we need to reuse `default-config.libsonnet` in both `deployments` and `operator`? Thanks @squat

Reply:
This change requires by https://github.com/observatorium/operator/pull/5

Reply:
looks good with https://github.com/observatorium/operator/pull/5 :+1: